pom.xml

```diff
@@ -281,6 +281,13 @@
     </dependency>
   </dependencies>
 
+  <repositories>
+    <repository>
+      <id>repository.sonatype.org</id>
+      <url>https://repository.sonatype.org</url>
```

Comment:
This should not be necessary. Configuration you your local `settings.xml` should handle this.

Reply:
Yeah, sorry, I added this as a test due to the builds failing on Jenkins. The logs seemed to indicate that it was attempting to pull the API dep from the jenkins repo. Likely this is something I'll have to work with Justin on to determine where the repos are configured for the Jenkins pipelines and how to access the snapshots.